mobile/screens/Resource/ResourceHomeScreen.js

```diff
@@ -31,7 +31,7 @@ const ResourceHomeScreen = ({ navigation, props }) => {
   const { width, height } = useWindowDimensions();
   const { userId } = useContext(AuthContext);
   const { globalResources, setGlobalResources } = useContext(ResourceContext);
-  const { selectedSettingsLanguage } = useContext(SettingsContext);
+  const {selectedSettingsLanguage } = useContext(SettingsContext);
```

Comment:
Nit: This is just cosmetic but why remove the space?

Reply:
I think I may have just accidentally edited it. I will revert.